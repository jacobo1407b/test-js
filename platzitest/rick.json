{
    "id":1,
    "name":"Rick",
    "status":"Alive",
    "specie:":"Human",
    "gender":"Male"
}