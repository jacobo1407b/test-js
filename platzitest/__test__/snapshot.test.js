import {getCharacter} from '../snapshot';
import rick from '../rick.json';


describe('es hora de las instantaneas', () => {
    test('Snapshot', () => {
       expect(getCharacter(rick)).toMatchSnapshot(); 
    });
});

