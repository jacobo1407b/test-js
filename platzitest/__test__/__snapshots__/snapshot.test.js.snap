// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`es hora de las instantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick",
  "species": undefined,
  "status": "Alive",
}
`;
